src/env.cc

```diff
@@ -917,6 +917,17 @@ void AsyncHooks::grow_async_ids_stack() {
 uv_key_t Environment::thread_local_env = {};
 
 void Environment::Exit(int exit_code) {
+  if (options()->trace_exit) {
+    HandleScope handle_scope(isolate());
+
+    fprintf(stderr,
+            "(node:%d) WARNING: Exited the environment with code %d\n",
```

Comment:
Since we’re also printing this for Workers, maybe it makes sense to also print `env->thread_id()` (or maybe only if it is a Worker, to avoid redundant information in the common case)?

Reply:
I was wondering if printing thread ids may be helpful since there is no public JS API to get it. Anyway, I like the idea.

Reply:
> there is no public JS API to get it.

There is `require('worker_threads').threadId`, unless we’re talking about different things?

Reply:
Sounds like I've been outdated. Updates coming.